src/ngraph/runtime/cpu/pass/cpu_assignment.cpp

```diff
@@ -473,6 +474,22 @@ namespace ngraph
                     }
                 }
 
+                template <>
+                void CPUAssignment::ASSIGN_DECL(ngraph::op::Reshape)
+                {
+                    auto reshape = static_cast<op::Reshape*>(node);
+
+                    if (node->get_input_element_type(0) == element::f32 &&
+                        node->get_input_shape(0).size() < 12 &&
```

Comment:
dont hardcode 12. instead use the macro value (https://github.com/intel/mkl-dnn/blob/master/include/mkldnn_types.h) from MKLDNN